Description.md

```diff
@@ -30,39 +30,73 @@ Spread this extension and help us raise environmental awareness! Let's preserve
 German (de)
 -----------
 
```

Comment:
@PrateekGupta1509, these translations have grammatical errors. Did you translate them yourself? Or did you use an automatic translator?

It would be great if you could add a translation to your native language. There is no need to try to translate the description to languages in which you are not a native speaker. It is better to leave them blank, until native speakers volunteer to translate to those languages.